### PR TITLE
Add few commands and Update GM Command

### PR DESCRIPTION
**Pull request by ghost**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/commands/open.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    -- validate target
+    local targ = player:getCursorTarget()
+    if targ == nil then
+        error(player, "A door NPC must be selected.")
+        return
+    end
+
+    targ:openDoor(open_time)
+
+
```
Comment (zach2good) [inline scripts/commands/open.lua]: Extra whitespace

File: scripts/commands/sanction.lua [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---------------------------------------------------------------------------------------------------
+-- func: Sanction
+-- desc: 
+---------------------------------------------------------------------------------------------------
+require("scripts/globals/besieged")
+
+
+cmdprops =
+{
+    permission = 0,
```
Comment (zach2good) [inline scripts/commands/sanction.lua]: If server operators want to let their players give themselves sanction/signet etc. at will, that's up to them, but it shouldn't be part of the commands available to players by default.

File: scripts/commands/spin.lua [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- func: spin <player>
+-- desc: Spins a character in place. Used for fishbot detection.
```
Comment (zach2good) [inline scripts/commands/spin.lua]: Could you expand the comment here and explain how this is used to observe/trap/detect fishbotters?

**Review by zach2good — CHANGES_REQUESTED** [review]
Hello and welcome 👋 

Please double-check the style guide in regards to whitespace, newlines and function spacing (`player:PrintToPlayer( "Sanction" )` etc.).
https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#lua

I think `open` and `spin` are fine, and the extra logging is probably also fine, but the permissions on sanction/signet/sigil seem off to me

File: scripts/commands/spin.lua (hunk 3 above)
Comment (ghost) [reply scripts/commands/spin.lua]: Suppose to turn them in different direction to help catch if doing any fishbot.

File: scripts/commands/open.lua (hunk 1 above)
Comment (ghost) [reply scripts/commands/open.lua]: That is how they had the command. I have tested the command and got no errors

Comment (zach2good) [reply scripts/commands/open.lua]: Who's _they_?

Comment (ghost) [reply scripts/commands/open.lua]: Twilight on Wings

Comment (zach2good) [reply scripts/commands/open.lua]: Gonna reply to this in the main thread

**Comment by zach2good** [comment]
> Legiongreed
> That is how they had the command. I have tested the command and got no errors
>
> zach2good
> Who's they?
>
> Legiongreed
> Twilight on Wings

Unfortunately, we're not willing to take code from Wings unless the original authors come and give it to us directly, specifically saying that it is under the license that we use (GPLv3) and not the license they use (AGPLv3). 

I won't go into license specifics or project history, but we have a much more stress-free existence when we keep to ourselves and don't try to mix licenses (since AGPL is functionally backwards-**in**compatible with GPL).

**Comment by TeoTwawki** [comment]
It pisses me off that this pr was even opened. The not-author then goes as far as deleting their own github account. This is exactly what I have executed to eventually happen. 

I am not going to be anywhere near as polite as Zach about this. Don't PR code that isn't under the projects license, don't PR code that belong to someone else without permission, and don't PR code that isn't properly attributed to who wrote it!!!

**Pull request closed without merging**